SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Base.cpp

```diff
@@ -354,6 +380,18 @@ std::vector< BaseData* > Base::findGlobalField( const std::string &name ) const
     return result;
 }
 
+/// Find fields given a name: several can be found as we look into the alias map
+std::vector< BaseDDGLink* > Base::findGlobalDDGLink( const std::string &name ) const
+{
+    std::vector<BaseDDGLink*> result;
+    //Search in the aliases
+    typedef MapDDGLink::const_iterator mapIterator;
```

Comment:
I think this is a good case for using auto : `auto range = m_aliasDDGLink.equal_range(name);`